packages/schematics/tsconfig.json

```diff
@@ -9,8 +9,6 @@
     "noFallthroughCasesInSwitch": true,
     "noImplicitAny": true,
     "noImplicitThis": true,
-    "noUnusedParameters": true,
```

Comment:
Why were these removed?

Reply:
I've removed them during the development process. I'll revert.